tests/cases/fourslash/completionsClassMembers.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path="fourslash.ts" />
+
+////interface I {
+////    method(): void;
+////}
+////
+////export class C implements I {
+////    property = "foo"
+////
+////    /**/
+////}
+
+verify.baselineCompletions();

```

Comment:
Can you also baseline completions when snippet completions and class snippet completions are both enabled?

Reply:
https://github.com/microsoft/TypeScript/blob/5b4a8d4134bbb8947abdb0f916568b2e1ff6b2c0/src/services/completions.ts#L1403-L1404
?